crates/solidity/testing/perf/src/tests/references.rs

```diff
@@ -17,16 +17,16 @@ pub fn run(binding_graph: BindingGraph) {
         }
         reference_count += 1;
 
-        let resolution = reference.resolve_definition();
-        if resolution.is_ok() {
+        let definitions = reference.definitions();
+        if !definitions.is_empty() {
             resolved_references += 1;
         }
     }
 
     assert_eq!(reference_count, 1652, "Failed to fetch all references");
 
     assert_eq!(
-        resolved_references, 1409,
+        resolved_references, 1490,
```

Comment:
I wonder why would the # of references go up in this case?

Reply:
Before we were checking if the reference resolved to a single definition. Now we're checking if it resolves to *any* number of definitions. This difference is the number of previously ambiguous resolution results.